modules/ROOT/examples/csharp/Program.cs

```diff
@@ -1186,4 +1212,39 @@ public void Log(LogLevel level, LogDomain domain, string message)
         }
     }
     // end::custom-logging[]
+
+    // tag::local-win-conflict-resolver[]
+    class LocalWinConflictResolver : IConflictResolver
+    {
+        Document Resolve(Conflict conflict)
+        {
+            return conflict.LocalDocument;
+        }
+    }
+    // end::local-win-conflict-resolver[]
+
+    // tag::remote-win-conflict-resolver[]
+    class RemoteWinConflictResolver : IConflictResolver
+    {
+        public Document Resolve(Conflict conflict)
+        {
+            return conflict.RemoteDocument;
+        }
+    }
+    // end::remote-win-conflict-resolver[]
+
+    // tag::merge-conflict-resolver[]
+    class MergeConflictResolver : IConflictResolver
+    {
+        public Document Resolve(Conflict conflict)
+        {
+            var localDict = conflict.LocalDocument.ToDictionary();
+            var remoteDict = conflict.RemoteDocument.ToDictionary();
+            var result = localDict.Concat(remoteDict)
+               .GroupBy(kv => kv.Key)
+               .ToDictionary(g => g.Key, g => g.First().Value);
+            return new MutableDocument(conflict.DocumentID, result);
+        }
```

Comment:
Oh very nice indeed, an implementation of merge. I had written a placeholder method [in swift](https://github.com/couchbase/docs-couchbase-lite/blob/release/2.6/modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift#L994) but it won't be of much help to users. I'll update it to have an implementation of merge as well.